test/xpu_api/utilities/function.objects/arithmetic.operations/modulus.pass.cpp

```diff
@@ -0,0 +1,60 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include "support/test_config.h"
+
+#include <oneapi/dpl/functional>
+
+#include "support/utils.h"
+
+#if TEST_DPCPP_BACKEND_PRESENT
+class KernelModulusTest;
+
+void
+kernel_test()
+{
+    sycl::queue deviceQueue = TestUtils::get_test_queue();
+    bool ret = false;
+    sycl::range<1> numOfItems{1};
+    sycl::buffer<bool, 1> buffer1(&ret, numOfItems);
+    deviceQueue.submit([&](sycl::handler& cgh) {
+        auto ret_access = buffer1.get_access<sycl::access::mode::write>(cgh);
+        cgh.single_task<class KernelModulusTest>([=]() {
+            typedef dpl::modulus<int> F;
+            const F f = F();
```

Comment:
Could you please apply previous fix here and broadcast in into all similar files:

* modulus.pass.cpp
* multiplies.pass.cpp
* negate.pass.cpp
* plus_arifm_op.pass.cpp
* bit_and.pass.cpp
* bit_not.pass.cpp
* bit_or.pass.cpp
* bit_xor.pass.cpp
* equal_to.pass.cpp
* greater.pass.cpp
* greater_equal.pass.cpp
* less.pass.cpp
* less_equal.pass.cpp
* not_equal_to.pass.cpp
* logical_and.pass.cpp
* logical_not.pass.cpp
* logical_or.pass.cpp
* binary_negate.pass.cpp
* unary_negate.pass.cpp

Reply:
Partially fixed.